packages/mask/.webpack/manifest.ts

```diff
@@ -3,18 +3,39 @@ import emitFile from '@nice-labs/emit-file-webpack-plugin'
 import { cloneDeep } from 'lodash-unified'
 import { NormalizedFlags } from './flags'
 
-type Manifest = typeof import('../src/manifest.json') & Record<string, any>
+type ManifestV2 = typeof import('../src/manifest.json') & { key?: string }
+type ManifestV3 = typeof import('../src/manifest-v3.json') & { key?: string }
+
 export function emitManifestFile(flags: NormalizedFlags) {
     return emitFile({
```

Comment:
suggest, use `emitJSONFile` replace it.

see https://github.com/NiceLabs/emit-file-webpack-plugin/blob/617c374a84b4f043ae15d087d2f3f999927b4a33/index.ts#L58-L65

Reply:
I don't know if we can do that. Some of our release processes may involve editing `../src/manifest.json`